README.md

```diff
@@ -98,55 +98,52 @@ en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
+> ***NOTE*** On macOS you need to explicitly publish ports since the `--network=host` option doesn't work there. Make sure that the ports you are publishing with the option `-p` of the `docker run` command are corresponding to the ports specified within the `INTEGRATED_TURN_PORT_RANGE` variable. Please also keep in mind that the port range cannot be too wide as it might cause problems with the docker container starting.
 
 Finally, go to <http://localhost:4000/>.
 
 ## More advanced configuration
 
 ### Environment variables
-Below you can find a list of runtime environment variables, used to configure the demo:
-```
-VIRTUAL_HOST={host passed to the endpoint config, defaults to "localhost" on non-production environments (MIX_ENV != prod)}
-
-USE_TLS={"true" or "false", if set to "true" then https will be used and certificate paths will be required}
-KEY_FILE_PATH={path to certificate key file, used when "USE_TLS" is set to true}
-CERT_FILE_PATH={path to certificate file, used when "USE_TLS" is set to true}
-
-EXTERNAL_IP= {the IP address, on which TURN servers will listen}
-INTEGRATED_TURN_PORT_RANGE={port range, where UDP TURN will try to open ports. By default set to 50000-59999. 
-    The bigger the range is, the more users server will be able to handle. Useful when not using the `--network=host` option to 
-    limit the UDP ports used only to ones exposed from a Docker container.}
-INTEGRATED_TCP_TURN_PORT={port number of TCP TURN}
-INTEGRATED_TLS_TURN_PORT={port number of TLS TURN, used when "INTEGRATED_TURN_PKEY" and "INTEGRATED_TURN_CERT" are provided}
-INTEGRATED_TURN_CERT={SSL certificate for TLS TURN}
-INTEGRATED_TURN_PKEY={SSL private key for TLS TURN}
-
-STORE_METRICS={"true" or "false", if set to "true", then `Membrane.RTC.Engine` metrics will be stored in the database. By default set to "false"}
-METRICS_SCRAPE_INTERVAL={number of seconds between `Membrane.RTC.Engine` metrics reports scrapes}
-
-DATABASE={name of database used to store `Membrane.RTC.Engine` metrics reports}
-DB_USERNAME={name of the database user}
-DB_PASSWORD={password for the database user}
-DB_HOSTNAME={hostname of the database}
-DB_PORT={port, where database will set up its endpoint}
-
+Below you can find a list of runtime environment variables, used to configure the application:
+* `VIRTUAL_HOST` - host passed to the endpoint config, defaults to `localhost` on non-production environments (where `MIX_ENV` != prod)
+
+* `USE_TLS` - `true` or `false`, if set to `true` then https will be used and certificate paths will be required
+* `KEY_FILE_PATH` - path to certificate key file, used when `USE_TLS` is set to true
+* `CERT_FILE_PATH` - path to certificate file, used when `USE_TLS` is set to true
+
+* `EXTERNAL_IP` - the IP address, on which TURN servers will listen
```

Comment:
I have added information about the default value of `EXTERNAL_IP` and pointed out that not setting it will cause the videoroom to be accessible only on localhost. I believe this is sufficient as we have almost the whole chapter on how to set up `EXTERNAL_IP`